llama_stack/configs/examples/local-router-run.yaml

```diff
@@ -0,0 +1,64 @@
+built_at: '2024-09-18T13:41:17.656743'
+image_name: local
+docker_image: null
+conda_env: local
+apis_to_serve:
+- inference
+- memory
+- safety
+- telemetry
+- agents
+- models
+provider_map:
+  inference: models-router
+  safety:
+    provider_id: meta-reference
+    config:
+      llama_guard_shield:
+        model: Llama-Guard-3-8B
+        excluded_categories: []
+        disable_input_check: false
+        disable_output_check: false
+      prompt_guard_shield:
+        model: Prompt-Guard-86M
+  telemetry:
+    provider_id: meta-reference
+    config: {}
+  agents:
+    provider_id: meta-reference
+    config: {}
+  models:
+    provider_id: builtin
```

Comment:
initial impression: this feels odd and should not be needed to be seen by the user. 

also config -> models_config should just be config